kpet/data.py

```diff
@@ -188,6 +218,26 @@ def matches(self, target):
                 return False
         return True
 
+    def validate(self, target):
+        """
+        Check if the pattern matches at least a single target
```

Comment:
This is not really a correct description. We only get a single target passed to this function.

Reply:
Hopefully better now